sysutils/apcupsd/Makefile

```diff
@@ -0,0 +1,8 @@
+PLUGIN_NAME=        apcupsd
+PLUGIN_DEVEL=       yes
+PLUGIN_VERSION=     0.1
+PLUGIN_DEPENDS=     apcupsd
+PLUGIN_COMMENT=     Apcupsd plugin for OPNsense
```

Comment:
```suggestion
PLUGIN_COMMENT=     APC UPS daemon
```

Reply:
Not sure I fully agree...   
This is a plugin for OPNSense to bridge between APCUPSD program (that is installed separately by plugin-installer) and OPNSense.   Yes, it is a APC UPS Daemon.  But that is also the separate plugin NUT that could be used with APC UPSes as well.  APCUPSD is more dedicated to APC UPSes.

Maybe a tradeoff:   
"APCUPSD - APC UPS daemon plugin for OPNsense."
or
"APCUPSD - APC UPS daemon plugin"

:-)

//Dan Lundqvist (mrzaz)

Reply:
Well people know it’s a plugin since they install from plugins tab in GUI. And plugins are integrations for services and daemons. I just want to avoid pushing redundant information.

Reply:
True. 
But I still think there should be a reference to the APCUPSD function that this build uppon. But have no issue dropping the plugin reference. :-)
So maybe "APCUPSD - APC UPS daemon" then?
//Dan Lundqvist (mrzaz)

Reply:
@mrzaz sure, changed the suggestion

Reply:
:thumbsup: